### DISCUSSION TITLE
Feature Request: Browse by topic / multitopic (ala Reddit's subreddits/multireddits)

### DISCUSSION THREAD

**Discussion by platform-kit** [dailydotdev/daily#763] [Feature Requests]
See title. It seems counterintuitive to not be able to sort news. Some days I want to only see what's happening in the Js/Node/Vue world. Other days I want to see what's happening in AI/ML. I currently have no way of sorting/sifting by topic (or custom groups of topics). It is very frustrating. Otherwise daily dev is awesome.

**Comment by idoshamun** [comment]
Hi @platform-kit,

Thanks for the suggestion. We call this feature custom feeds and there's an existing discussion here: https://github.com/dailydotdev/daily/issues/336
